modules/appendix/examples/ldbc_snb/select_statement_index_three_post_accum_clauses.gsql

```diff
@@ -4,14 +4,14 @@ INTERPRET QUERY () {
     SumAccum<INT> @cnt2;
     SumAccum<INT> @@global_t_count;
 
-    R   =  SELECT s
-        FROM Person:s-(Likes>) -:msg - (Has_Creator>)-Person:t
+    R = SELECT s
+        FROM (s:Person) -[Likes>]- :msg -[Has_Creator>]- (t:Person)
```

Comment:
```suggestion
        FROM (s:Person) -[:Likes]-> (:msg) -[:Has_Creator]-> (t:Person)
```
Notice 
(1) How the arrows move from inside the brackets to outside
(2) This is a 2-hop path. `edge` is a vertex type in the middle of the path. We need parentheses around it.